aya/src/util.rs

```diff
@@ -143,6 +144,17 @@ macro_rules! include_bytes_aligned {
     }};
 }
 
+pub(crate) fn page_size() -> usize {
+    // Safety: libc
+    (unsafe { sysconf(_SC_PAGESIZE) }) as usize
+}
+
+// bytes_of converts a <T> to a byte slice
+pub(crate) unsafe fn bytes_of<T>(val: &T) -> &[u8] {
```

Comment:
The type T should have a Pod bound, and then it could be safe since Pod guarantees that T can be a slice of bytes

Reply:
I can't do that unless I also `unsafe impl Pod for bpf_map_def`, `unsafe imple Pod for bpf_insn` etc...
Happy to do it if that works for you... or we can defer it to a cleanup of the `Pod` trait with a default `as_bytes` impl

Reply:
Oh yeah sorry I didn't think about that. Yeah no need to do that now